biz.aQute.bndlib/src/aQute/bnd/differ/Baseline.java

```diff
@@ -204,13 +204,19 @@ public boolean contains(Diff diff) {
 					content = pdiff.getDelta();
 					break;
 
-				case REMOVED :
+				case REMOVED : // this is a package removal not a member removal
```

Comment:
How can an exported package removal be a micro change? If it has been moved to another bundle then this should be solved in another place?

This is bundle versioning which I always recommended not to take too seriously. What problem are we solving here?